src/app/Common/DAL/FluentUpdater/Updater.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Common.DAL.Documents;
+
+namespace Common.DAL.FluentUpdater
+{
+    public static class Updater<TDocument> where TDocument : BaseDocument
```

Comment:
@ilyahryapko Do not get the problem this class is solving. Does it simply wrap `MongoFluentUpdater` in order to mitigate usage of `new` operator in the repository calls?

```
await _userRepository.UpdateOneAsync(id, Updater<User>
   .Set(u => u.PasswordHash, newPassword.GetHash())
   .Set(u => u.ResetPasswordToken, string.Empty));
```
vs
```
await _userRepository.UpdateOneAsync(id, **new** MongoFluentUpdater<User>()
   .Set(u => u.PasswordHash, newPassword.GetHash())
   .Set(u => u.ResetPasswordToken, string.Empty));
```

Reply:
Yep. Simply because of that. "Updater" looks cleaner then new object with long name